src/System.Collections.Concurrent/ref/System.Collections.Concurrent.csproj

```diff
@@ -5,6 +5,7 @@
     <AssemblyVersion>4.0.10.0</AssemblyVersion>
```

Comment:
I bumped the version (though, I'm not confident I changed it in all the right places). Tests pass locally, but CI is now failing across the board (previously was green) and I'm not sure what the issue is. Hmm.